doc/contributors_guide/storage_plugins_mongodb-atlas.md

```diff
@@ -0,0 +1,33 @@
+Mongodb Storage plugin
+===============
+
+Quick Guide
+-------
+
+This is a quickguide for users wishing to connect to a mongodb atlas server. It is assumed that they have access 
+to a server and have their own username and password. 
+
+```python
+    >>> user = "USERNAME"
+    >>> password = "PASSWORD"
+    >>> server = "MYSERVERADDRESS"
+    >>> uri = f"mongodb+srv://{server}"
```

Comment:
We have to think how to handle the "mongodb+srv" scheme correctly. The issue is that the URL passed to functions like `dlite.Instance.from_url()` is split up internally by dlite into four components

    <SCHEME>://<LOCATION>?<OPTIONS>#<ID>

where the <SCHEME> part is used to identify the plugin. But we cannot create a plugin named `mongodb+srv` since it is not a valid Python identifier. I don't think this is an issue for the plugins written in C, since the plugin name is never used a s an identifier in that case. 

I am not sure what would be the best solution. One possibility could be that the C API that loads the Python plugins, strips off the "+" and everything that follows it, when looking for a driver and then provides the full scheme as an optional argument.

Reply:
I have made a solution that should be handle all of these cases, see the latest commit